lib/serialisers/json-0.6.js

```diff
@@ -120,7 +120,7 @@ module.exports = createClass({
     }
 
     if (element.content) {
-      return [this.serialise(element.content)];
+      return this.serialiseContent(element.content);
```

Comment:
This might be `[this.serialiseContent(element.content)]`. But I am not confident in either of them.

Reply:
We should be wrapping the content in an array, because in API Elements 1.0 (the `Element` object), content should be an element. In API Elements 0.6 (what we are serialising to), the content should be an array. Thus the content should be wrapped in an array.